src/lightbox/LightboxFooter.js

```diff
@@ -35,7 +36,7 @@ export default class LightboxFooter extends PureComponent<Props> {
     const { displayMessage, onOptionsPress, style } = this.props;
     return (
       <View style={[styles.wrapper, style]}>
-        <Text style={styles.text}>{displayMessage}</Text>
+        <Label style={styles.text} text={displayMessage} />
```

Comment:
Also this is customises message, which is constructed using stream name, which should not be translated (actually not possible.)
But we need to translate `shared in/with you` text.
https://github.com/zulip/zulip-mobile/blob/8e112271a40135a429c2fcb3eb84fccd14caa712/src/lightbox/LightboxContainer.js#L89